graylog2-server/src/main/java/org/graylog2/indexer/Deflector.java

```diff
@@ -207,7 +207,12 @@ public int getNewestTargetNumber() throws NoTargetIndexException {
         return Collections.max(indexNumbers);
     }
 
-    public String[] getAllDeflectorIndexNames() {
+    /**
+     * Returns a list of all Graylog managed indices.
+     *
+     * @return list of managed indices
+     */
+    public String[] getAllIndexNames() {
```

Comment:
Maybe `getAllGraylogIndexNames` would be more specific (and would also match `isGraylogIndex`).